src/session.cc

```diff
@@ -80,10 +80,30 @@ sysfail::ActivePlan::ActivePlan(const Plan& p) : p(p) {
     }
 }
 
+void unmask_sigsys(int signum) {
+    struct sigaction sa;
+    // Retrieve current signal action
+    if (sigaction(signum, NULL, &sa) != 0) {
+        sysfail::log("Failed to get signal handler for signal %d: %s\n", signum, strerror(errno));
+        return;
```

Comment:
is this actually ever supposed to fail?  (if we filter out SIGSTOP and SIGKILL out)

if not, perhaps we should throw an exception or use an `assert(ret == 0);`?